include/expr/Parser.h

```diff
@@ -230,7 +230,8 @@ namespace expr {
     /// expressions.
     static Parser *Create(const std::string Name,
                           const llvm::MemoryBuffer *MB,
-                          ExprBuilder *Builder);
+                          ExprBuilder *Builder,
+			  bool QueryIndependence);
```

Comment:
When you make a new PR or do a force push you can fix this. You can extract the patch, apply it to your working directory and then use ``git clang-format`` to reformat only your changes and then commit.